.env.example

```
@@ -7,4 +7,4 @@ AI_API_KEY=your_ai_api_key
 
 JWT_EXPIRATION=3600000
 
-CORS_ALLOWED_ORIGIN=http://localhost:3000
\ No newline at end of file
+CORS_ALLOWED_ORIGINS=http://localhost:3000
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)